pkg/streamingpromql/functions.go

```diff
@@ -22,6 +23,7 @@ type InstantVectorFunctionOperatorFactory func(
 	annotations *annotations.Annotations,
 	expressionPosition posrange.PositionRange,
 	timeRange types.QueryTimeRange,
+	innerExpressions parser.Expressions,
```

Comment:
[nit] Perhaps `argumentExpressions` would be a clearer name? Or `argExpressions` to maintain symmetry with `args`?